.flake8-base

```diff
@@ -30,7 +30,7 @@ exclude =
     ./wandb/proto_check/
     ./wandb/env.py
     ./wandb/integration/magic.py
-    ./wandb/integration/keras/
+    ./wandb/integration/keras/keras.py
```

Comment:
Let's lint the legacy callback in a future PR, but do lint the newly added stuff.